build.gradle

```diff
@@ -563,6 +579,48 @@ project('test:system') {
 
         onlyIf { dockerRegistryUrl && repoUrl }
     }
+
+    task execShellScript(type: Exec) {
+        commandLine "./preTestScript.sh"
+        args "$CLUSTER_NAME", "$MASTER", "$NUM_SLAVES"
+    }
+
+    //This is  used to invoke docker based systemTests
+    task startSystemTestsWithDocker(type: Test) {
+
+            dependsOn 'testJarForDocker', 'execShellScript'
+
+        ext.dockerRegistryUrl = project.hasProperty("dockerRegistryUrl") ? project.dockerRegistryUrl : ""
+
+        description 'Used to invoke docker based system tests, example usage: gradle startSystemTests -DmasterIP=xx.xx.xx.xx'
+        testClassesDir = sourceSets.test.output.classesDir
+        classpath = sourceSets.test.runtimeClasspath
+
+        systemProperty "execType", System.getProperty("execType")
+        systemProperty "masterIP", System.getProperty("masterIP")
+        systemProperty "imageVersion", System.getProperty("imageVersion")
+        systemProperty "skipServiceInstallation", System.getProperty("skipServiceInstallation")
+        systemProperty "segmentStoreExtraEnv", System.getProperty("segmentStoreExtraEnv")
+
+
+        if (System.getProperty("execType") == null) {
+            systemProperty "execType", "REMOTE_SEQUENTIAL"
```

Comment:
we should change the default for gradle task `startSystemTestsWithDocker`

Reply:
done.